ruby/red-adbc.gemspec

```diff
@@ -52,7 +52,7 @@ Gem::Specification.new do |spec|
   spec.files += Dir.glob("lib/**/*.rb")
   spec.extensions = ["dependency-check/Rakefile"]
 
-  spec.add_runtime_dependency("red-arrow")
+  spec.add_runtime_dependency("red-arrow", "<11.0.0")
```

Comment:
Could you revert this?
This is used by users not test.

Reply:
Unfortunately, I couldn't get CI to pass without it - one part of the tests is to install the gem

Reply:
As this is a known outstanding issue we can also just wait for the conda-forge PR to merge, and merge PRs in the meantime ignoring this issue. I can repurpose this PR to just switch to system meson

Reply:
Especially since the upstream PR just merged, so we can wait a couple hours and see. Guess I was a little too eager here.

Reply:
Ah, OK. Can we try the following? This will install old `red-arrow` when system Arrow GLib is old.

```diff
diff --git a/ci/scripts/glib_test.sh b/ci/scripts/glib_test.sh
index 51d955b..987cb17 100755
--- a/ci/scripts/glib_test.sh
+++ b/ci/scripts/glib_test.sh
@@ -74,6 +74,14 @@ test_subproject() {
         gem_flags='-- --with-cflags="-D_LIBCPP_DISABLE_AVAILABILITY" --with-cppflags="-D_LIBCPP_DISABLE_AVAILABILITY"'
     fi
 
+    arrow_glib_version=$(pkg-config --modversion arrow-glib | sed -e 's/-SNAPSHOT$//g' || :)
+    if [ -n "${arrow_glib_version}" ]; then
+        gem install \
+            --install-dir "${build_dir}/gems" \
+            --version ${arrow_glib_version} \
+            red-arrow \
+            -- ${gem_flags}
+    fi
     gem install --install-dir "${build_dir}/gems" pkg/*.gem -- ${gem_flags}
     popd
 }
```

Reply:
Thanks! Done. I also reverted the other changes and fixed up the Debian Dockerfile.